AcadeMeData/ForumModels.py

```diff
@@ -0,0 +1,23 @@
+from UserModels import *
+
+
+class MessageBoards(models.Model):
+    ID = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+    CourseName = models.CharField(max_length=30)
+    Rate = models.CharField(max_length=1, choices=range(1-5), default=3)
```

Comment:
`rating` may be a better name

Reply:
and do we want to have a default rating of 3? seems like it is better be defaulted as None